src/main/java/org/broadinstitute/consent/http/service/DacService.java

```diff
@@ -121,6 +121,15 @@ public DacService(DacDAO dacDAO, UserDAO userDAO, DataSetDAO dataSetDAO,
         return dacToUserMap;
     }
 
+    public List<Dac> findDacsByUser(AuthUser authUser) {
+        if (isAuthUserAdmin(authUser)) {
+            List<Dac> allDacs = dacDAO.findAll();
+            return allDacs;
+        }
+        List<Dac> dacs = dacDAO.findDacsForEmail(authUser.getGoogleUser().getEmail());
```

Comment:
The auth user's name is their email address, so this also works:
```suggestion
        List<Dac> dacs = dacDAO.findDacsForEmail(authUser.getName());
```